great_expectations/compatibility/azure.py

```diff
@@ -6,6 +6,11 @@
     "azure blob storage components are not installed, please 'pip install azure-storage-blob azure-identity azure-keyvault-secrets'"
 )
 
+try:
+    import azure
```

Comment:
Is this used?  If not, then maybe we do not need to import it?

Reply:
Good call, removed 🙇